master/contrib/Dockerfile

```diff
@@ -64,13 +64,13 @@ from ubuntu:12.04
 # Install buildbot and its dependencies
 
 run /bin/echo -e "\
-    deb http://archive.ubuntu.com/ubuntu precise main universe\n\
-    deb http://archive.ubuntu.com/ubuntu precise-updates main universe" > \
+    deb http://archive.ubuntu.com/ubuntu trusty main universe\n\
+    deb http://archive.ubuntu.com/ubuntu trusty-updates main universe" > \
     /etc/apt/sources.list
 run apt-get update
 run DEBIAN_FRONTEND=noninteractive apt-get install -y python-pip python-dev \
     supervisor git sudo ssh
-run pip install sqlalchemy==0.7.9 buildbot buildbot_slave
+run pip install -v sqlalchemy==0.7.9 buildbot buildbot_slave
```

Comment:
@shanzi I just wanted to note, that your changes not only updates Ubuntu image version, but also changes verbosity. Initially I thought that you added `-v` in commit by mistake.
If `-v` doesn't add too much output and works as "progress bar" for end user, then it's ok.